examples/match-active-domain/README.md

```diff
@@ -0,0 +1,256 @@
+# Example: Match Active Domain
+
+Use this as a guideline for utilizing the Wildlink Library's getDomains functionality.
+
+## Get Started
+
+1.  Install dependencies:
+
+    ```
+    yarn install
+    ```
+
+1.  Insert the Secret and Application ID provided by Wildlink into `index.js` on Line 21.
+
+    ```js
+      initialize = async () => {
+        try {
+          logger('Initializing Application');
+
+          const client = new WildlinkClient('SECRET', 0); // <---- Here
```

Comment:
I'm just doing it to be nitpicky, but maybe you can indicate in your commented out note (<---- Here) that the 0 is supposed to be replaced with the Application ID? I think when I initially saw it, there was a brief moment when I thought the 0 was supposed to stay hard coded. 

Other than that, super clean code, easy to understand what it's doing, and really nice move turning the array into a map.